source/parser/x86_64/x86_64.cpp

```diff
@@ -158,7 +111,7 @@ namespace smeagle::x86_64 {
         st::Type *paramType = param->getType();
 
         // Get register class based on type
-        std::pair<RegisterClass, RegisterClass> regClasses = getRegisterClassFromType(paramType);
+        Class c = getRegisterClassFromType(paramType);
```

Comment:
yeah this line is what threw me of... "A struct called Class?!"

Reply:
It's like Java!